pkg/internalapis/cnsvolumeoperationrequest/v1alpha1/cnsvolumeoperationrequest_types.go

```diff
@@ -55,6 +53,17 @@ type CnsVolumeOperationRequestStatus struct {
 	LatestOperationDetails []OperationDetails `json:"latestOperationDetails,omitempty"`
 }
 
+type QuotaDetails struct {
+	// Reserved keeps a track of the quantity that should be reserved in
+	// storage quota during a create/expand volume operation.
```

Comment:
We'll need this for create volume snapshots in the future.

Reply:
Updated the doc string